File: multiply.py
# Python : Calculate the multiplication of an array

def mutliplier(arr) :
    total = 1
    for num in arr:
        total *= num
    return total
 
numbers = [1, 2, 3, 4, 5]
result = mutliplier(numbers)
print("Multiplication of an array python :", result)

File: multiply.js
// JavaScript : Calculate the multiplication of an array
function multiplier(arr) {
	let total = 1;
	for (let num of arr) {
		total *= num;
	}
	return total;
}

let numbers = [1, 2, 3, 4, 5];
let result = multiplier(numbers);
console.log("Multiplication of Array in JavaScript:", result);
